### PR TITLE
webinspector: add type hints

### PR DESCRIPTION
**Pull request by guysalt**

### CODE REVIEW

File: pymobiledevice3/services/webinspector.py [hunk 1]
```diff
@@ -197,22 +197,22 @@ def open_app(self, bundle: str, timeout: Union[float, int] = 3) -> Application:
...
-    async def send_socket_data(self, session_id, app_id, page_id, data):
+    async def send_socket_data(self, session_id: str, app_id: str, page_id: int, data: Mapping):
         await self._forward_socket_data(session_id, app_id, page_id, data)
 
-    async def setup_inspector_socket(self, session_id, app_id, page_id):
+    async def setup_inspector_socket(self, session_id: str, app_id: str, page_id: int):
         await self._forward_socket_setup(session_id, app_id, page_id, pause=False)
 
-    def find_page_id(self, page_id):
+    def find_page_id(self, page_id: str) -> Optional[Tuple[Application, Page]]:
```
Comment (matan1008) [inline pymobiledevice3/services/webinspector.py]: Should not be optional

File: pymobiledevice3/services/webinspector.py [hunk 2]
```diff
@@ -197,22 +197,22 @@ def open_app(self, bundle: str, timeout: Union[float, int] = 3) -> Application:
...
             for page in self.application_pages[app_id]:
                 if page == page_id:
                     return self.connected_application[app_id], self.application_pages[app_id][page_id]
 
-    def flush_input(self, duration=0):
+    def flush_input(self, duration: Union[float, int] = 0):
         return self.await_(asyncio.sleep(duration))
 
-    def await_(self, awaitable):
+    def await_(self, awaitable: callable):
```
Comment (matan1008) [inline pymobiledevice3/services/webinspector.py]: awaitable is not callable

File: pymobiledevice3/services/webinspector.py (hunk 1 above)
Comment (guysalt) [reply pymobiledevice3/services/webinspector.py]: but if the page_id doesn't exist?

File: pymobiledevice3/services/webinspector.py (hunk 2 above)
Comment (guysalt) [reply pymobiledevice3/services/webinspector.py]: removed

File: pymobiledevice3/services/webinspector.py (hunk 1 above)
Comment (guysalt) [reply pymobiledevice3/services/webinspector.py]: done

**Review by matan1008 — APPROVED** [review]

**Review by doronz88 — APPROVED** [review]

**Pull request merged**